apps/designer/app/designer/features/workspace/canvas-tools/outline/outline.tsx

```diff
@@ -1,6 +1,8 @@
 import { useMemo } from "react";
 import { styled } from "@webstudio-is/design-system";
 
+type Rect = Pick<DOMRect, "top" | "left" | "width" | "height">;
```

Comment:
I think we should have this inside some reusable place, because we seem to need it everywhere

Reply:
Also, couldn't decide on the place :)

Reply:
same?